### ISSUE TITLE
remove numpy requirement

### ISSUE THREAD

**Issue by wuhu** [lf1-io/padl#317]
Currently numpy is used
- to initialize the seed in the dataloader
- to initialize a matrix in `Compose._pd_longrepr`
- in `wrap.py` to check if something is an instance of `np.ufunc`

We can probably remove all these dependencies without too many issues and thus remove numpy from the requirements.txt.

**Comment by sjrl** [comment]
I would say this is probably a low priority because `pytorch` has `numpy` as a dependency https://github.com/pytorch/pytorch/blob/master/requirements.txt
